reference-guide/configuration.adoc

```diff
@@ -1,4 +1,328 @@
 = Configuration Reference
+:icons: image
+:iconsdir: ../images/icons
 include::../partials/attributes.adoc[]
 
-Please see the full https://docs.kogito.kie.org/latest/html_single/#chap-kogito-configuring[configuration reference] at the Kogito Documentation page.
\ No newline at end of file
+== {PRODUCT} Configuration Properties
+
+[.configuration-legend]
+icon:lock[title=Fixed at build time] Configuration property fixed at build time - All other configuration properties are overridable at runtime
+[.configuration-reference.searchable, cols="80,.^10,.^10"]
+|===
+h|[[kie-flyway_configuration]]link:#kie-flyway_configuration[Configuration property]
```

Comment:
@LightGuard this looks a bit weird, why we use kie-flyway_configuration in this header anchor? I'd say that the  ke-flyway config could go inside of this table. (kie.flyway.enabled: true|false)